src/models/model_loader.cpp

```diff
@@ -212,28 +386,10 @@ bool ModelLoader::create(const tue::config::DataConstPointer& data, const UUID&
     // Set type
     req.setType(id, type);
 
-    geo::Pose3D pose = geo::Pose3D::identity();
-
     // Get pose
-    if (r.readGroup("pose"))
-    {
-        r.value("x", pose.t.x);
-        r.value("y", pose.t.y);
-        r.value("z", pose.t.z);
-
-        double roll = 0, pitch = 0, yaw = 0;
-        r.value("X", roll,  tue::config::OPTIONAL);
-        r.value("Y", pitch, tue::config::OPTIONAL);
-        r.value("Z", yaw,   tue::config::OPTIONAL);
-        r.value("roll",  roll,  tue::config::OPTIONAL);
-        r.value("pitch", pitch, tue::config::OPTIONAL);
-        r.value("yaw",   yaw,   tue::config::OPTIONAL);
-
-        // Set rotation
-        pose.R.setRPY(roll, pitch, yaw);
-
-        r.endGroup();
-    }
+    geo::Pose3D pose = geo::Pose3D::identity();
+    if (!ed::models::readPose(r, pose))
+        error << "[ed::models::create] No pose, while reading model: '" << id << "'" << std::endl;
 
     pose = pose_offset * pose;
```

Comment:
Continuing computing with poses even if you cannot read it?

Reply:
Not having a pose isn't really an error.